changelogs/unreleased/5124-add-support-collect-metrics.yml

```diff
@@ -0,0 +1,6 @@
+description: Add support to write collected metrics to the database
+issue-nr: 5124
+change-type: minor
+destination-branches: [master, iso6]
+sections:
+  minor-improvement: "{{description}}"
```

Comment:
I wouldn't add a changelog entry for each sub-ticket in this epic, but only a single one at the end of the epic.
```suggestion
```